docs/docs.md

```diff
@@ -0,0 +1,585 @@
+      })[]
+  auto: (
+    | {
+        attempts: {
+          max: number
+          avg: number
+        }
+        successes: {
+          max: number
+          avg: number
+        }
+        statName: string
+      }
+    | {
+        // total
+        attempts: number
+        // total
+        successes: number
+        statName: string
+      })[]
+}[]
+```
+
+# `/event/{eventKey}/team/{team}/info`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  // only if they have future matches
+  nextMatch?: {
+    redAlliance: string[]
+    blueAlliance: string[]
+    // UTC date - match predicted time
+    time: string
+    // example: qm3
+    key: string
+  }
+  rank: number
+  // ranking score
+  rankingScore: number
+}
+```
+
+# `/event/{eventKey}/team/{team}/stats/auto`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  modeName: string
+  stats: (
+    | {
+        // 2018orwil
+        event: string
+        // qm1
+        match: string
+        attempts: number
+        successes: number
+        statName: string
+      }
+    | {
+        // 2018orwil
+        event: string
+        // qm1
+        match: string
+        attempted: boolean
+        succeeded: boolean
+        statName: string
+      })[]
+}[]
+```
+
+# `/event/{eventKey}/team/{team}/stats/teleop`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = (
+  | {
+      // 2018orwil
+      event: string
+      // qm1
+      match: string
+      attempts: number
+      successes: number
+      statName: string
+    }
+  | {
+      // 2018orwil
+      event: string
+      // qm1
+      match: string
+      attempted: boolean
+      succeeded: boolean
+      statName: string
+    })[]
+```
+
+# `/event/{eventKey}/teams`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = string[]
+```
+
+# `/events`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  name: string
+  district?: string
+  week?: number
+  // UTC date
+  startDate: string
+  // UTC date
+  endDate: string
+  location: {
+    lat: number
+    lon: number
+  }
+}[]
+```
+
+# `/schema`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  teleop: {
+    statName: string
+    statKey: string
+    type: "number" | "boolean"
+  }[]
+  auto: {
+    statName: string
+    statKey: string
+    type: "number" | "boolean"
+  }[]
+}
+```
+
+# `/team/{team}/automodes`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = string[]
+```
+
+# `/team/{team}/stats/auto`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  modeName: string
+  stats: (
+    | {
+        // 2018orwil
+        event: string
+        // qm1
+        match: string
+        attempts: number
+        successes: number
+        statName: string
+      }
+    | {
+        // 2018orwil
+        event: string
+        // qm1
+        match: string
+        attempted: boolean
+        succeeded: boolean
+        statName: string
+      })[]
+}[]
+```
+
+# `/team/{team}/stats/teleop`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
```

Comment:
Why does /teleop send data for both teleop and auto? Comparing this endpoint (`/team/{team}/stats/teleop`) to `/event/{eventKey}/team/{team}/stats/teleop`, the /event one specifies what event each data point comes from but this one doesn't which seems backwards to me. Also, what is the use case for grabbing all the data on a team across all events?